### PR TITLE
Mobile rank

### PR DESCRIPTION
**Pull request by jread-usgs**
for #234 

I have some of lindsay's commits in here and probably need to resolve conflicts. 

I also don't know how to tap into the "isMobile", so I created a variable to use in the meantime (we'd replace `rankSvg.mobile` w/ a function call or something else. 
image

I don't think what we have right now for this fig is usable on mobile.

### CODE REVIEW

File: js/rank_states_utils.js [hunk 1]
```diff
@@ -179,18 +191,29 @@ function rankEm(barData) {
     }
     rankSvg.updateStyles();
   };
-  
+  var width = rankSvg.desktopWidth;
+  var height = rankSvg.desktopHeight;
+  if (rankSvg.mobile){
```
Comment (lindsaycarr) [inline js/rank_states_utils.js]: I think you can grab `waterUseViz.mode` which will be either `desktop` or `mobile`. That's a global var, so it should be available here, too.

**Comment by lindsaycarr** [comment]
Wow, Jordan! Jumping ahead and already implementing a solution to #201! Yep, I was going to work on this very issue today. This looks good to me. A little confused on the order in which things should get merged since this has some of my PR in it.

**Comment by aappling-usgs** [comment]
I just merged Lindsay's PR and see that there are no merge conflicts here, which is a relief. I guess the other possibility is that something from Lindsay's PR might be unintentionally overwritten by this PR? Jordan's branch for this PR is 6 commits ahead, 7 commits behind USGS-VIZLAB:master...my guess is that it's all good, but Lindsay, maybe you could read over the changes one more time to make sure nothing happened?

Given that Jordan is at the TRAC meeting today, I'd be happy to create commits and a second PR that takes care of using `waterUseViz.mode` instead of `rankSvg.mobile` after we get this PR in.

**Comment by aappling-usgs** [comment]
@jread-usgs , does "I don't think what we have right now for this fig is usable on mobile." mean you think this is _less_ usable than the desktop version we had yesterday, or just that it's no more usable than that one was? If the former, what is different that needs to be addressed?

**Comment by lindsaycarr** [comment]
Agree, I'm fine with moving ahead on this without Jordan implementing requested changes and having one of us do them. I am slightly concerned that it is going to write over the commits I just added to these files that aren't shown in this PR. Would it be easiest to close this and just copy the small amount of code that actually made this change?

**Comment by jread-usgs** [comment]
for this `I don't think what we have right now for this fig is usable on mobile.` I meant what we have up on beta. Sorry, was unclear.

**Comment by lindsaycarr** [comment]
Ok and with that, Jordan has merged from upstream and it looks all fine. Let's merge this and then deal with `waterUseViz.mode`.

**Pull request merged**